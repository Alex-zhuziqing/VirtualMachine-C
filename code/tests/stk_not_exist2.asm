FUNC LABEL 0
    MOV STK A VAL 55
    PRINT PTR A
    RET
