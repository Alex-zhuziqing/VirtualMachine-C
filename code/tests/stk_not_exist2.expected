Stack symbol 0 does not exist in Stack 1.
